docs/source/collect/configmap.md

```diff
@@ -59,7 +76,11 @@ When this collector is executed, it will include the following file in a support
   "key": "password",
   "configMapExists": true,
   "keyExists": true,
-  "value": "mypass"
+  "value": "mypass",
+  "data:": {
```

Comment:
should you add `includeAllData: true` after line 50?

Reply:
yes